base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x, T)
+    mod(x, T)
+    %(x, T)
+
+For an integer type `T`, finds `z::T` such that `x` ≡ `z` (mod n), where n is the
+cardinality of `T` and `z` is restricted to the interval of values representable by `T`.
```

Comment:
Given that both terminologies ("cardinality of a type", "a type's native modular arithmetic") are potentially challenging, perhaps describing this behavior in *both* ways (to help readers tripped up by one or the other expressions) *and* including an example (e.g. that above) would be worthwhile? E.g., "For an integer of type `T`, finds [... present version in the PR ...]. In other words, finds `z::T` such that `x ≡ z` in the native modular arithmetic of type `T`. For example, `x & Int8` returns [...]"?